Makefile

```diff
@@ -1,6 +1,19 @@
 # Read project name from .env file
+
+SYSTEM=$(shell uname)
+
+ifeq ($(SYSTEM),Windows)
+WINDOWS=1
+COMPOSE_ARGS="-d"
+$(shell cp -n .env.default .env)
+$(shell cp -n ./src/docker/docker-compose.override.yml.default ./src/docker/docker-compose.override.yml)
+else
+COMPOSE_ARGS=""
+WINDOWS=0
```

Comment:
This is cosmetic but I would have moved "WINDOWS=0" on top of "COMPOSE_ARGS" to keep it like for the "windows" case (line 6)